### ISSUE TITLE
error on running main_il.py

### ISSUE THREAD

**Issue by HongminWu** [shikharbahl/neural-dynamic-policies#1]
Would you upload a data sample for running main_il.py? 
the error is 
# file = '/home/sbahl2/research/digit_generator/arr.m'?

**Comment by csufangyu** [comment]
hi,I get the same erroe.Have you fixed it?

**Comment by shikharbahl** [comment]
Hi, 

Sorry about that. I got rid of the dependency in `smnist_loader.py`: https://github.com/shikharbahl/neural-dynamic-policies/commit/6c8afe3b1d66850d76edb2bb7f401a0580c77d17#. However, the array file is too large to share on the repo, so I have provided a downloadable link here: https://drive.google.com/file/d/1oqJBbczrURrGmOU37J3TSCAkwSbqeXzG/view?usp=sharing. Note that you must add the path to this data file in the smnist loader.
